src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -312,6 +317,26 @@ public async Task SynchronizeWithServer_StandaloneMode_UIContextIsCleared()
             toolWindowServiceMock.Invocations.Count.Should().Be(0);
         }
 
+        [TestMethod]
```

Comment:
One new test to check that the taint synchronizer fetches and passes on the expected branch name.